app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java

```diff
@@ -286,6 +287,6 @@ private void assertNoProductsFound(Response<Search> response) {
         Search search = response.body();
         List<Product> products = search.getProducts();
         assertTrue(products.isEmpty());
-        assertTrue(Integer.valueOf(search.getCount()) == 0);
+        Assert.assertEquals(0, (int) Integer.valueOf(search.getCount()));
```

Comment:
Just import `assertEquals` statically

Reply:
Will do that @VaiTon

Reply:
Done. Can you please review it again @VaiTon